### ISSUE TITLE
encourage use of get_fit tables

### ISSUE THREAD

**Issue by fluidpixel** [NUKnightLab/sql-mysteries#11]
Currently you don't need to access the get fit tables in order to find the solution.
<details><summary>spoiler alert...</summary>
<p>
The event that both witnesses attended only has one other person checkin on that day, which is the killer. If the event had a few (many?) more check-ins it might encourage digging into the interviews and therefore gym details...
</details>

**Comment by JoeGermuska** [comment]
yes, I figured that out as I was creating an "instructor's guide."   I think it'd be worth adding some noise.

**Comment by Khelavaster** [comment]
Same goes for the second part of the puzzle.

<details>
 <summary>spoiler</summary>
<p>
If you work backwards and select people who went to see the concert three times in specified timespan, it returns two people. But only one of those people has a driver's license, which ends the search right there. Also judging from the name, the other person is a man, which also ends the search.

It would be useful to have more matches for the concert goers, so that the car and other details are relevant to the search.
</p>
</details>

**Comment by JoeGermuska** [comment]
if people want to cook up some SQL statements that help increase noise, we'd welcome that.

**Comment by Khelavaster** [comment]
Sure thing. Where do you want them checked in?
